docs/DeveloperGuide.md

```diff
@@ -6,9 +6,6 @@
 
 ## Design & implementation
 
-{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
-
-
 ## Product scope
```

Comment:
I guess your team hasn't mentioned which target group your application focuses on and the value proposition of your application. Please add that so that users can know the purpose of your application.